packages/components/src/tabs/tab.tsx

```diff
@@ -18,6 +18,12 @@ import {
 import type { WordPressComponentProps } from '../context';
 import { chevronRight } from '@wordpress/icons';
 
+/**
+ * Renders a single tab.
+ *
+ * The currently active tab receives default styling that can be
+ * overridden with CSS targeting `[aria-selected="true"]`.
+ */
```

Comment:
As discovered elsewhere in a similar PR, this duplication of JSDoc is currently unavoidable 😓

One is for IDE IntelliSense, the other is for the docgen.

Reply:
Never mind, I forgot I wrote a crawler to forcibly retrieve this information from the AST 😂 bb01937489fcdd0a65d9eabbcba8338d2acde2e4